live-build/config/hooks/configuration/template.ctl

```diff
@@ -6,7 +6,7 @@ Standards-Version: 3.9.2
 Package: delphix-entire-@@PLATFORM@@
 Provides: delphix-entire
 Version: @@VERSION@@
-Extra-Files: packages.list, variant
+Extra-Files: packages.list, variant, hotfix_metadata
```

Comment:
Can we come up with a filename that better describes the contents of the file.. e.g. perhaps `fixed-hotfixes.list` or perhaps just `hotfixes.list`? since the contents is a list of all the hotfixes that have been fixed in this upgrade image that is being generated, and the `.list` suffix would be consistent with the `packages.list` file.